quartz/plugins/transformers/ofm.ts

```diff
@@ -143,6 +145,8 @@ const wikilinkImageEmbedRegex = new RegExp(
   /^(?<alt>(?!^\d*x?\d*$).*?)?(\|?\s*?(?<width>\d+)(x(?<height>\d+))?)?$/,
 )
 
+const inlineFootnoteRegex = /\^\[((?:[^\[\]]|\[(?:[^\[\]]|\[[^\[\]]*\])*\])*)\]/g
```

Comment:
Can you also 	add regex explanation here as well.

Reply:
I tried 🫡